apis/nucleus/src/index.js

```diff
@@ -482,6 +485,23 @@ function nuked(configuration = {}) {
       getRegisteredTypes: types.getList,
       __DO_NOT_USE__: {
         types,
+        popover(anchorElement, fieldIdentifier, options = { show: true }) {
+          if (api._popoverInstance) {
+            root.remove(api._popoverInstance);
+            api._popoverInstance = null;
+          }
```

Comment:
Son now when there is an existing pop-over instance, we remove it and create a new one. I thought we wanted to keep the pop-over instance, and replace the contents (i.e. a different field and position).

What is the difference between this and just mounting and unmounting the popover everytime it opens and closes?

Reply:
yes, I wanted to keep it, but ran into all sorts of issues. So made it this way for now, with the intention of revisiting. Bascially the API should be stable, but we ca change the implementation